src/pages/_app.tsx

```diff
@@ -162,36 +164,11 @@ function MyApp({ Component, pageProps }: AppPropsWithLayout) {
 	// used to poll for scwAddress in "waitForScwAddress"
 	const scwAddressRef = useRef(scwAddress)
 
-	const getUseNonce = useCallback(async() => {
+	const getUseNonce = useCallback(async(): Promise<string> => {
```

Comment:
Why'd we need the extra type here?

Reply:
To have a better looking, cleaner code

Reply:
this is making things verbose I feel, Intellisense already tells us it's a string